lib/pbench/server/hierarchy.py

```diff
@@ -73,6 +73,11 @@ def check_controller_in_list(self, controller):
             ]
         )
 
+    def _output_bad_controllers(self, fp):
+        fp.write("\nUnexpected files found:\n")
+        for controller in sorted(self.bad_controllers):
+            fp.write(f"\t{os.path.basename(controller)}\n")
+
```

Comment:
Is _output_bad_controllers() used? I don't see a call in this commit.

Reply:
it is being used in controller_hierarchy dump and users_hierarchy dump